website/content/en/highlights/2021-11-16-failed-event-routing.md

```diff
@@ -0,0 +1,107 @@
+---
+date: "2021-11-16"
+title: "Failed event routing feature released"
```

Comment:
```suggestion
title: "Initial support for routing failed events released"
```

Example of updating the language a bit to reflect that this is a beta feature under active development.